sys/include/net/ng_ipv6/netif.h

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (C) 2014 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    net_ng_ipv6_netif IPv6 network interfaces
+ * @ingroup     net_ng_ipv6
+ * @brief       IPv6 specific information on @ref net_ng_netif
+ * @{
+ *
+ * @file
+ * @brief       Definitions for IPv6 specific information of network interfaces.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+#ifndef NG_IPV6_NETIF_H_
+#define NG_IPV6_NETIF_H_
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#include "kernel_macros.h"
+#include "kernel_types.h"
+#include "mutex.h"
+#include "net/ng_ipv6/addr.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @def NG_IPV6_NETIF_ADDR_NUMOF
+ *
+ * @brief   Number of IPv6 addresses per interface.
+ */
+#ifdef MODULE_NG_IPV6_ROUTER
+#define NG_IPV6_NETIF_ADDR_NUMOF    (5) /* router needs all-routers multicast address */
+#else
+#define NG_IPV6_NETIF_ADDR_NUMOF    (4)
```

Comment:
Can you explain me the rational? Right now, if I start for instance, sixlowapp a node has already four link local addresses plus the loopback address:
```
            inet6 addr: fe80::ff:fe00:af1b/128  scope: local
            inet6 addr: ff02::1:ff00:af1b/128  scope: local [multicast]
            inet6 addr: ff02::1/128  scope: local [multicast]
            inet6 addr: ::1/128  scope: global
            inet6 addr: ff02::1:ff00:1/128  scope: local [multicast]
```
Which of them are not required?